src/ModuleList/Article/ArticleStyles.ts

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+
+export const MarkdownWrapper = styled('h3')`
+  color: white;
+  width: 80%;
+  line-height: 1.5rem;
+  p {
+    margin-bottom: 1rem;
+  }
+`;
+
+export const HeadingContainer = styled('h1')`
+  color: white;
+  width: 80%;
+  margin-bottom: 1rem;
+
+`;
+
+export const AuthorContainer = styled('h1')`
+  color: white;
+  width: 80%;
+  margin-bottom: 1rem;
+
+`;
+
+
+export const Div = styled('div')`
```

Comment:
Lets try and change this to something else? Possibly `Container`?
```suggestion
export const Container = styled('div')`
```